hedera-node/hedera-addressbook-service-impl/src/test/java/com/hedera/node/app/service/addressbook/impl/test/handlers/NodeGetInfoHandlerTest.java

```diff
@@ -221,7 +221,7 @@ private NodeInfo getExpectedInfo(boolean deleted) {
                 .build();
     }
 
-    private Query createGeNodeInfoQuery(final long nodeId) {
+    private Query createGetNodeInfoQuery(final long nodeId) {
```

Comment:
A bit of a nit, but this really should be createNodeGetInfoQuery since NodeGetInfo is what it's called everywhere else. I'm going to go ahead and approve this PR anyway, this can be fixed later.

Reply:
I fixed it